### PR TITLE
Rework the bubble-tea scene files to provide a more consistent experience

### LINKED ISSUE #10: Scene views inconsistent behaviour
### What happened?

Currently some scenes show some inconsistent and unwanted behavior. To track progress and see a list of issues, see #11.

### Version

v.0.1.3 (815c8a3c57979335d4f543c1759a90c99a6b2398)

### What OS are you seeing the problem on?

Linux

### What architecture are you seeing the problem on?

None

### Relevant log output

_No response_

### Any extra info:

_No response_

### PR DESCRIPTION
**Pull request by samox73**
fixes #10 

- [x] Add button mapping for quit/back on all scenes
- [x] Follow symlinks when creating a new installation
- [x] Show custom help bindings for the `mods` scene
- [x] Refactor `scene` pkg to several smaller, independent subpackages
- [x] Text input has unavailable key hints (e.g. new installation shows "j"/"k" as aliases to ArrowDown/ArrowUp even though j/k are caught by the input field
- [x] Various `AdditionalShortHelpKeys`/`AdditionalFullHelpKeys` are not shown correctly (or are obsolete)

### CODE REVIEW

File: tea/scenes/apply.go [hunk 1]
```diff
@@ -149,13 +150,13 @@ func (m apply) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
...
-		case KeyControlC:
+		case keys.KeyControlC:
 			return m, tea.Quit
-		case KeyEscape:
+		case keys.KeyEscape:
 			m.cancelled = true
 			m.cancelChannel <- true
 			return m, nil
-		case KeyEnter:
+		case keys.KeyEnter:
```
Comment (SoMuchForSubtlety) [inline tea/scenes/apply.go:159]: bubbletea provides constants for key codes[^1]. Simply switch on `msg.Type`.

[^1]: https://pkg.go.dev/github.com/charmbracelet/bubbletea#KeyType

**Review by SoMuchForSubtlety — CHANGES_REQUESTED** [review]

**Comment by budak7273** [comment]
Fixes #16

**Review by Vilsol — DISMISSED** [review]

**Comment by Vilsol** [comment]
Please sync your PR to latest master @samox73

**Review by Vilsol — APPROVED** [review]

**Comment by budak7273** [comment]
Thanks for your contributions!

**Pull request merged**